src/custom_nodes/add_one/Dockerfile.ubuntu

```diff
@@ -16,6 +16,7 @@
 
 FROM ubuntu:20.04
 RUN apt update && apt install -y build-essential curl
+RUN curl -s https://download.01.org/opencv/master/openvinotoolkit/thirdparty/linux/opencv/opencv_4.5.2-076_ubuntu20.txz | tar --use-compress-program=xz -xvf -
```

Comment:
We should not add opencv downloading step to node building that does not require opencv

Reply:
Maybe separate opencv_utils.hpp from utils.hpp?